layouts/partials/menu.html

```diff
@@ -26,6 +26,21 @@
         {{ end }}
       {{ end }}
     {{ end }}
+
+    {{ if $.Site.Params.showLanguageSelector }}
+    <div class="spacer"></div>
```

Comment:
You don't have to use `spacer` element. Just use flexbox styles for the parent element with:

```
display: flex;
justify-items: space-between;
```

Reply:
This would then space all elements even in the row. I don't think that this looks pretty.

![Screenshot_2019-08-05 Rinma dev — It's simple until you make it complicated](https://user-images.githubusercontent.com/1405824/62487552-753d4980-b7c2-11e9-8a24-5c428b4435d9.png)

Reply:
Oh, right... I thought that there're only two elements (children)...